server/controllers/temporary-accommodation/manage/bookingsController.ts

```diff
@@ -53,16 +54,30 @@ export default class BookingsController {
 
       const premises = await this.premisesService.getPremises(callConfig, premisesId)
       const room = await this.bedspacesService.getRoom(callConfig, premisesId, roomId)
+      try {
+        const person = await this.personsService.findByCrn(callConfig, crn)
 
-      setUserInput(req)
+        setUserInput(req)
 
-      return res.render('temporary-accommodation/bookings/confirm', {
-        premises,
-        room,
-        crn,
-        arrivalDate,
-        departureDate,
-      })
+        return res.render('temporary-accommodation/bookings/confirm', {
+          premises,
+          room,
+          person,
+          arrivalDate,
+          departureDate,
+        })
+      } catch (err) {
+        if (err.status === 404) {
+          insertGenericError(err, 'crn', 'doesNotExist')
+        }
```

Comment:
Should this also handle `403`, same as `create`?

Reply:
oops yes, I forgot to add that. thanks!